### DISCUSSION TITLE
Centering an image not working

### DISCUSSION THREAD

**Discussion by EctoBoggan** [requarks/wiki#4921] [Error / Bug Report]
Hi,
first sorry for my English,
This is how I center an image when writing an article in visual editor mode : 

1. click on "insert assets" (the little picture icon)
2. upload my image if not already uploaded
3. select the image i want to insert
4. select "centered" from the drop down menu
5. click on "insert"

But this is not working, it ends up being left aligned.

Am I doing something wrong ?
Thank you for you futur answer and this project !

**Comment by ghost** [comment]
Same problem here.

Which drop down menu are you talking about, because the options below image do not have a "centered" option.

![image](https://user-images.githubusercontent.com/5992169/170202271-ffa8d61e-bbec-45d1-b001-b6750943b00b.png)

This option seems out of place here. An "align center" option would be better instead.

**Comment by EctoBoggan** [comment]
I was talking about this menu : 
![image](https://user-images.githubusercontent.com/54527880/170203282-20bace0d-7c6d-4cd6-8a3b-d1dc5a420bcf.png)

you arrive on this menu after clicking the image button : 
![image](https://user-images.githubusercontent.com/54527880/170203479-85f0383f-b1a8-4378-8460-aa9bb77e0c2b.png)

But since no one answered me and I couldn't find any solution for this + got some anoying bugs, I just kept using Bookstack instead of WikiJS.

Have a great day.

**Reply by ghost** [reply]
That's a shame. I hope Bookstack suits your needs.

As for image alignment, I don't think that asset management should be the place for such an option. Practically in every editor image alignment happens while editing a page.
